src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,28 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_USER_AGE = 18;
+    private static final int MIN_USER_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User is null");
+        }
+        if (user.getAge() == null || user.getAge() < MIN_USER_AGE) {
+            throw new RuntimeException("Invalid User age value");
```

Comment:
```suggestion
            throw new RuntimeException("Invalid User age value: " + user.getAge());
```
Optional in every throw could be added a data that is wrong